Extension/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 1.1.4: Janurary 15, 2021
+### Bug Fix
+* Disable the Insiders update channel for Mac versions less than 18 due to bug [#6787](https://github.com/microsoft/vscode-cpptools/issues/6787).
```

Comment:
We would probably use the 10.x version string here, right?  But I'm not sure that we need to ship this yet.

Reply:
Yeah, good catch. Keep getting the darwin versus regular versions confused.

Reply:
...maybe turn this disabling on/off via a cpptools.json option...